docs/baseband/mark5b/index.rst

```diff
@@ -4,8 +4,97 @@
 MARK 5B
 *******
 
-The Mark 5B VLBI format is described in its `design specifications
-<http://www.haystack.mit.edu/tech/vlbi/mark5/mark5_memos/019.pdf>`_.
+The Mark 5B format is the output format of the Mark 5B disk-based VLBI data
+system.  It is described in its `design specifications
+<http://www.haystack.mit.edu/tech/vlbi/mark5/mark5_memos/019.pdf>`_ (these
+exact specifications are also found in the `Mark 5B user manual
+<https://www.haystack.mit.edu/tech/vlbi/mark5/docs/Mark%205B%20users%20manual.pdf>`_).
+
+.. _mark5b_file_structure:
+
+File Structure
+==============
+
+Each :term:`data frame` consists of a :term:`header` consisting of four 32-bit
+words (16 bytes) followed by a :term:`payload` of 2500 32-bit words (10000
+bytes).  The header contains a sync word, frame number, and timestamp 
+(accurate to 1 ms), as well as user-specified data; see Sec. 1 of the
+documentation for details.  The payload supports 1, 2, 4, 8, 16 and 32-bit
+samples, with the oldest sample in a word occupying the least significant
+bit.  Files start with a header (unlike for Mark 4), and an integer number of
+frames fit within 1 second.
+
+The Mark 5B system also outputs files wiht the active bit-stream mask, number
+of frames per second, and observational metadata.  Baseband does not use these
```

Comment:
Aside: interesting to hear about these extra files. I've never seen them...

Reply:
I'll make a note of where to find this info in their specifications.